src/binary-criteria/README.md

```diff
@@ -0,0 +1,23 @@
+# Binary Criteria Search Engine
+
+/!\ Read carefully
+
+This test is aimed to see how you search through data, how you manage async calls, and undefined criteria depth.
+
+- The `criteria.spec.ts` file should not be modified at all, as well as `criteria.json`, `criteria2.json` and `data.sample.json`.
```

Comment:
peut être wrap dans un `describe('do not modify', () => {});` pour autoriser la personne à ajouter des tests